### PR TITLE
Added grafana plugins

### PR DESCRIPTION
**Pull request by Ab-hishek**
## What does this PR do?
This PR adds some of the grafana plugins and also adds some Prometheus configuration relabeling for kube-state-metrics to collect all labels info from all the pods that it collects metrics from.

Grafana plugins that are added are:
* grafana-polystat-panel(polystat plugin)
* snuids-trafficlights-panel(traffic-light plugin)

### How to achieve this - > We should to be able to filter out openebs specific pods to visualise in grafana. For that we can make use of `kube_pod_labels` metric coming from `kube-state-metrics` and do a join operation along with other metrics.

* In order to list pods with all their labels in Prometheus query we need to do a join operator( group clause) with the required metric and `kube_pod_labels`(which list all labels of all pods).
Example: `kube_pod_info * on (pod,namespace) group_right kube_pod_labels`
                 This would list pods with all its labels
* In order to filter out pods with specific labels we need to apply that labels inside the prometheus query itself.
Example: `kube_pod_info * on (pod,namespace) group_right kube_pod_labels{label_openebs_io_component_name="ndm-operator"}`
                This would list only those pods that having the `openebs.io/component-name=ndm` label 

## Screenshots for this PR are attached below:
1. List pods with all their labels
Screenshot from 2021-05-20 14-28-07

2. List pods with specific labels only
Screenshot from 2021-05-20 14-27-51

3.  New grafana plugin added
Screenshot from 2021-05-20 13-45-27

Signed-off-by: Abhishek Agarwal <abhishek.agarwal@mayadata.io>

### CODE REVIEW

File: deploy/charts/openebs-monitoring/values.yaml [hunk 1]
```diff
@@ -89,6 +89,22 @@ kube-prometheus-stack:
...
+    serviceMonitor:
+      # relabel configs to apply to samples before ingestion.
+      ##
+      # - sourceLabels: [__meta_kubernetes_pod_node_name]
+      #   separator: ;
+      #   regex: ^(.*)$
+      #   targetLabel: nodename
+      #   replacement: $1
+      #   action: replace
+      relabelings:
```
Comment (rajaSahil) [inline deploy/charts/openebs-monitoring/values.yaml]: How should we configure this for installing openebs-monitoring in custom Prometheus stack if `servicemonitor` for `kube-state-metrics` is not installed?

Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/values.yaml:104]: This is not necessary. Without this also will work. I'll be removing this.

Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/values.yaml]: This is not required and will be removing it. The relabeling is not required. The filtration feature needed by us can simply be accomplished by using metrics join. Will update the PR description accordingly also.

**Review by rajaSahil — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**